samples/attested_tls/server/enc/identity_verifier.cpp

```diff
@@ -23,17 +23,17 @@ oe_result_t enclave_identity_verifier_callback(
         TLS_SERVER "identity->security_version = %d\n",
         identity->security_version);
 
-    // the unique ID for the enclave, for SGX enclaves, this is the MRENCLAVE
-    // value
+    // the unique ID for the enclave, for SGX enclaves, this is client's
+    // MRENCLAVE
     printf(TLS_SERVER "identity->unique_id(MRENCLAVE) :\n");
     for (int i = 0; i < OE_UNIQUE_ID_SIZE; i++)
-        printf(TLS_SERVER "0x%0x ", (uint8_t)identity->unique_id[i]);
+        printf("0x%0x ", (uint8_t)identity->unique_id[i]);
```

Comment:
But we are not doing a check to see that it is a known value, is that correct

Reply:
We are not checking for client enclave's MRENCLAVE because a TLS server enclave was built before an TLS client enclave. So, checking MRSIGNER is the most we could do unless we want to provision allowed client MRENCLAVE values after a server is running.